data/src/main/java/org/oppia/android/data/backends/gae/api/PlatformParameterService.kt

```diff
@@ -8,15 +8,14 @@ import retrofit2.http.Query
 interface PlatformParameterService {
 
   /**
-   * This function returns a Map of Platform Parameters where the keys corresponds to parameter
-   * names and values corresponds to their server value.
-   * [Reference](https://github.com/oppia/oppia/blob/develop/core/controllers/platform_feature.py)
+   * Retrieves map of platform parameters where the keys corresponds to parameter names and values
+   * corresponds to their server value.
+   * [Reference](https://github.com/oppia/oppia/blob/50f5cffdfccf19dad4d7144addb44a74cad23d3c/core/controllers/platform_feature_test.py)
```

Comment:
Did you mean to change this to point to the test file? That's not expected, I think, since tests don't technically define the actual contract of a component's interface.

Reply:
No I didn't meant to point to test file. My mistake. Changing it to actual code file. (https://github.com/oppia/oppia/blob/50f5cffdfccf19dad4d7144addb44a74cad23d3c/core/controllers/platform_feature.py)